params/network_upgrades.go

```diff
@@ -156,18 +165,22 @@ func (n *NetworkUpgrades) GetAvalancheRules(time uint64) AvalancheRules {
 }
 
 // getDefaultNetworkUpgrades returns the network upgrades for the specified avalanchego upgrades.
-// These should not return nil values.
+// Nil values are used to indicate optional upgrades.
 func getDefaultNetworkUpgrades(agoUpgrade upgrade.Config) NetworkUpgrades {
 	return NetworkUpgrades{
 		SubnetEVMTimestamp: utils.NewUint64(0),
 		DurangoTimestamp:   utils.TimeToNewUint64(agoUpgrade.DurangoTime),
 		EtnaTimestamp:      utils.TimeToNewUint64(agoUpgrade.EtnaTime),
-		FortunaTimestamp:   utils.TimeToNewUint64(agoUpgrade.FortunaTime),
+		FortunaTimestamp:   nil, // Fortuna is optional and have no effect on Subnet-EVM
 	}
 }
 
 // verifyWithDefault checks that the provided timestamp is greater than or equal to the default timestamp.
 func verifyWithDefault(configTimestamp *uint64, defaultTimestamp *uint64) error {
+	if defaultTimestamp == nil {
+		return nil
+	}
+
```

Comment:
supernit maybe move this below the configTimestamp nil check so it matches arguments order

Reply:
Ordering matters here since both values are nil.